src/Chapter2.hs

```diff
@@ -741,7 +762,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains n l = map (\x -> if n `elem` x then x else []) l
```

Comment:
In this case, you would end up with a bunch of empty lists in the result (which were not present in the initial input).
What you can do instead is to use `filter` function, to actually just leave the "right" lists 🙂